### ISSUE TITLE
Does the App class must be inherited directly from "Xamarin.Forms.Application" class?

### ISSUE THREAD

**Issue by IbrahimElshafey** [MicrosoftDocs/xamarin-docs#443]
I'm trying to use live reload with a project created from "Xamarin Prism" template where "App" is inherited from "PrismApplication" class, but live reload doesn't work.

With default Xamarin project template it works fine.

**Comment by davidbritch** [comment]
Hi @IbrahimElshafey 

Prism's `PrismApplicationBase` class inherits from `Application`, so shouldn't cause any problems with Live Reload. In fact, Live Reload does work with Prism, provided that you follow the Live Reload instructions (https://docs.microsoft.com/xamarin/xamarin-forms/xaml/live-reload).

However, also be aware that Live Reload is still a preview release, and as such you may encounter issues in using it at the moment.

**Comment by IbrahimElshafey** [comment]
Hi David Britch,

Thanks so mush for your help. I finally solved the problem by chance when I
put `LiveReload.Init();` after ` InitializeComponent();`  like the
following code

*InitializeComponent();*
* SetPrefernces();*
* await
NavigationService.NavigateAsync($"MasterDetailPage/NavigationPage/{nameof(InsuranceMainPage)}");*
*#if DEBUG*
*            LiveReload.Init();*
*#endif*

If I put `LiveReload.Init();` before `InitializeComponent();` it will not
work.

It's a great tool that I waited a long time. Thanks :)
